lib/xboxkrnl/xboxkrnl.h

```diff
@@ -132,12 +132,7 @@ typedef enum _MODE {
 /**
  * This struct defines a counted string used for ANSI-strings
  */
-typedef struct _ANSI_STRING
-{
-    USHORT Length;        /**< Length in bytes of the string stored in the buffer */
-    USHORT MaximumLength; /**< Length in bytes of the buffer */
-    PSTR Buffer;          /**< Pointer to the buffer used for the character-string */
-} ANSI_STRING, *PANSI_STRING, STRING, *PSTRING, OBJECT_STRING, *POBJECT_STRING;
+typedef STRING OBJECT_STRING, *POBJECT_STRING;
```

Comment:
The comment for this is broken.

Reply:
Should that typedef get moved, too?

Reply:
The object API is part of the kernel; I'm not sure if anything else uses it.

I think this is a custom name for Xbox anyway - I believe I checked Windows SDK and DDK and wasn't able to find this name? Current MSDN documents the users as `UNICODE_STRING` (which would be equivalent to a generic `ANSI_STRING`?) as far as I know.

Also I had no use for it as part of `OutputDebugString`, so I didn't want to touch it.